iothub_client/tests/longhaul_resource_cleanup/longhaul_resource_cleanup.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef LONGHAUL_RESOURCE_CLEANUP_H
+#define LONGHAUL_RESOURCE_CLEANUP_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* LONGHAUL_RESOURCE_CLEANUP_H */
```

Comment:
curious the point of this file?

Reply:
linux-options fails without it. Same goes with the other longhaul tests https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_build/results?buildId=79981&view=logs&j=9157c21d-e235-5a95-355e-08c3ffae2e82